scripts/qa_crowbarsetup.sh

```diff
@@ -5583,9 +5583,14 @@ function reboot_controller_clusters
             wait_for 400 5 \
                 "ssh $m_hostname 'if \`which drbdadm &> /dev/null\`; then drbd-overview; ! drbdadm dstate all | grep -v UpToDate/UpToDate | grep -q .; fi'" \
                 "drbd devices to be consistent on node $m_hostname"
+            ssh $m_hostname crm --wait node standby
+            wait_for 30 5 "ssh $m_hostname crm status | grep -q 'Node $m_hostname: standby'" "node $m_hostname to enter crm standby mode"
```

Comment:
Are you sure `$m_hostname` is properly evaluated in the grep command?

Reply:
Yes, single quotes inside double quotes are literals.